### PR TITLE
docs: add Telstra to compatibility chart

### PR DESCRIPTION
**Pull request by fillwithjoy1**
Add Telstra

Originally I was going to add AldiMobile, but considering it takes from Telstra's network it's probably best to say Telstra works as that would include all MVNOs

### CODE REVIEW

**Comment by fillwithjoy1** [comment]
Backup Calling wasn't working on my Nokia X30 5G, but it could be a handset issue

**Comment by fillwithjoy1** [comment]
Backup Calling wasn't working on my Nokia X30 5G, but it could be a handset issue

**Comment by fillwithjoy1** [comment]
Do I need to set the MNC code to 01, Telstra which is what the MVNO takes the network of?

File: README.en.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Carriers which aren't possible for testing by developer but reported as supporte
 
 #### Optional
 
-only if installing Schizuku using ADB
```
Comment (kyujin-cho) [inline README.en.md:40]: Nice catch!

**Review by kyujin-cho — APPROVED** [review]

File: README.en.md (hunk 1 above)
Comment (fillwithjoy1) [reply README.en.md:40]: thank you!

**Pull request merged**